src/NUnitFramework/tests/Constraints/DictionaryContainsKeyConstraintTests.cs

```diff
@@ -189,7 +190,7 @@ public void ShouldCallContainsKeysMethodOnPlainDictionary()
         {
             var dictionary = new TestNonGenericDictionary(99);
 
-            Assert.Catch<ArgumentException>(() => Assert.That(dictionary, Does.ContainKey(99)));
+            Assert.That(dictionary, Does.ContainKey(99));
```

Comment:
Should we also have a line
```csharp
Assert.That(dictionary, !Does.ContainKey(35));
```
so that the test have the same structure as `ShouldCallContainsKeysMethodOnDictionary`

Reply:
We should, thanks for catching this. I've just pushed an update for this